lua/glance/lsp.lua

```diff
@@ -8,8 +8,7 @@ local function create_handler(method)
       bufnr,
       method.lsp_method,
       params,
-      function(err, m, results)
-        results = method.lsp_method == m and results or m
+      function(err, result, _)
```

Comment:
The handler parameters is `err`, `result`, `cxt` and `config`. See [location handler](https://github.com/neovim/neovim/blob/29b80f6f2e9391b5d78390f65d09f00f73829310/runtime/lua/vim/lsp/handlers.lua#L359-L389)